snake/snake_game_khr.py

```diff
@@ -0,0 +1,113 @@
+from time import sleep
+import os
+
+
+class Board:
+    def __init__(self, width, height, top_border="<", bottom_border=">", vert_border="^"):
+        self.top_boarders = top_border
+        self.bottom_boarders = bottom_border
+        self.vert_boarders = vert_border
+        self.width = width
+        self.height = height
+        self.board = self.new_board()
+
+    def new_board(self):
+        new_board = []
+        for i in range(self.height):
+            if i in {0}:
```

Comment:
==

Reply:
@mykytapavlov 
If I do this, top border will be removed (I've tried this, by the way)

Reply:
if i == 0 ? (seems to work fine)